includes/class-amp-story-post-type.php

```diff
@@ -890,20 +890,35 @@ public static function get_embed_template( $template, $type, $templates ) {
 
 	/**
 	 * Outputs a card of a single AMP story.
+	 *
 	 * Used for a slide in the Latest Stories block.
 	 *
-	 * @param WP_Post $post The AMP story post.
-	 * @param string  $size The size of the image.
-	 * @return void
+	 * @param array $args {
+	 *     The arguments to create a single story card.
+	 *
+	 *     @type WP_Post post The post in which to search for the featured image.
+	 *     @type string  size The size of the image.
+	 *     @type bool    disable_link Whether to disable the link in the <a> wrapping the card.
+	 * }
 	 */
-	public static function the_single_story_card( $post, $size ) {
-		$thumbnail_id = get_post_thumbnail_id( $post );
-		if ( ! $thumbnail_id ) {
+	public static function the_single_story_card( $args ) {
+		$args = wp_parse_args(
+			$args,
+			array(
+				'post'         => 0,
```

Comment:
I'd probably use `null` as the default here and then immediately after the `$args` part do `get_post( $args['post'] )` and bail early if `! $post`.

Reply:
Ah, good idea. I'll fix this soon.

Reply:
@swissspidy, sorry for the delay. https://github.com/ampproject/amp-wp/pull/2077/commits/a29e26df26c916e3eda81d02fe3da07df315730e uses a default of `null`, like you suggested.